spec/support/disability_compensation_form/submissions/with_everything.json

```diff
@@ -191,7 +191,69 @@
             }
           ]
         }
-      ]
+      ],
+      "toxicExposure": {
+        "gulfWar1990": {
+          "iraq": true,
+          "kuwait": true,
+          "qatar": true
+        },
+        "gulfWar1990Details": {
+          "iraq": {
+            "startDate": "1991-03-01",
+            "endDate": "1992-01-01"
+          },
+          "qatar": {
+            "startDate": "1991-02-12",
+            "endDate": "1991-06-01"
+          },
+          "kuwait": {
+            "startDate": "1991-03-15"
+          }
```

Comment:
Will not providing an `endDate` for Kuwait or Laos potentially be an issue in the future?

Reply:
each startDate or endDate is optional for Veterans and nullable in LH afaik

Reply:
No, in the case of these date ranges, both the start and the end dates are optional

Reply:
uh, what Christine said 😂. I mustn't've hit refresh on the PR before commenting